src/meta/model_v2/migration/src/m20240506_112555_subscription_partial_ckpt.rs

```diff
@@ -166,14 +106,18 @@ impl MigrationTrait for Migration {
 #[derive(DeriveIden)]
 enum Subscription {
     Table,
-    // delete
+    // common
+    SubscriptionId,
```

Comment:
Nits, actually we can re-use `crate::m20240304_074901_subscription.rs`,  `crate::m20240417_062305_subscription_internal_table_name.rs` and only define the new schema for table `subscription` here to make it more clear. We can change it at any time later since it wouldn't cause any backward compatibility issues.